packages/wrangler/src/cloudchamber/delete.ts

```diff
@@ -0,0 +1,79 @@
+import { startSection, error, endSection, cancel } from "@cloudflare/cli";
+import { inputPrompt } from "@cloudflare/cli/interactive";
+import { logDeployment, pickDeployment } from "./cli/deployments";
+import { DeploymentsService } from "./client";
+import { handleFailure, interactWithUser, loadAccountSpinner } from "./common";
+import { wrap } from "./helpers/wrap";
+import type { CommonYargsOptions } from "../yargs-types";
+import type {
+	CommonCloudchamberConfiguration,
+	CloudchamberConfiguration,
+	inferYargsFn,
+} from "./common";
+import type { Argv } from "yargs";
```

Comment:
We try not to use `Argv` directly from Yargs since it doesn't include top-level Wrangler flags. Instead, `CommonYargsArgv` from `yargs-types.ts` should be used where possible

Reply:
Agreed! Will change.